telescope/src/main/java/com/mattprecious/telescope/TelescopeLayout.java

```diff
@@ -627,6 +629,7 @@ static Handler getBackgroundHandler() {
         final int width = displayMetrics.widthPixels;
         final int height = displayMetrics.heightPixels;
 
+        @SuppressLint("WrongConstant")
```

Comment:
Lint complains for `PixelFormat.RGBA_8888` not being a valid constant here. Not sure what would be the appropriate replacement and how it worked so far.